### PR TITLE
Feature/cloud sql

### PR DESCRIPTION
**Pull request by PePoDev**

### CODE REVIEW

File: modules/sql/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------- | --------------- | :------: |
+| activation_policy | This specifies when the instance should be active. Can be either ALWAYS, NEVER or ON_DEMAND.                                                                                                             | `string`            | `"ALWAYS"`      |    no    |
+| auto_size         | Configuration to increase storage size automatically. Note that future terraform apply calls will attempt to resize the disk to the value specified in disk_size - if this is set, do not set disk_size. | `bool`              | `true`          |    no    |
+| availability_type | This specifies whether a PostgreSQL instance should be set up for high availability (REGIONAL) or single zone (ZONAL).                                                                                   | `string`            | `"ZONAL"`       |    no    |
```
Comment (3d-pro) [inline modules/sql/README.md:26]: Only for PostgreSQL?

File: modules/sql/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+| disk_type         | The type of data disk: PD_SSD or PD_HDD.                                                                                                                                                                 | `string`            | `"PD_SSD"`      |    no    |
+| instance_name     | instance name to create in cloud sql                                                                                                                                                                     | `string`            | n/a             |   yes    |
+| labels            | A set of key/value user label pairs to assign to the instance.                                                                                                                                           | `map(string)`       | `null`          |    no    |
+| machince_type     | Type of instance to provision database                                                                                                                                                                   | `string`            | `"db-f1-micro"` |    no    |
```
Comment (3d-pro) [inline modules/sql/README.md:36]: typo on `machine_type`

File: modules/sql/instance.tf [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  default     = "mysql"
+}
+
+variable "database_version" {
+  description = "version of database to install (see also https://cloud.google.com/sql/docs/sqlserver/db-versions)"
+  type        = string
+  default     = "5.7"
+}
+
+variable "machince_type" {
```
Comment (3d-pro) [inline modules/sql/instance.tf]: typo `machine_type`

File: modules/sql/README.md (hunk 1 above)
Comment (PePoDev) [reply modules/sql/README.md:26]: The availability type of the Cloud SQL instance, high availability (REGIONAL) or single zone (ZONAL).

Comment (PePoDev) [reply modules/sql/README.md:26]: Changed description to above.

**Review by 3d-pro — APPROVED** [review]

**Comment by 3d-pro** [comment]
wait for @oatenter

**Review by oatenter — COMMENTED** [review]
Reformatting the structure files

**Pull request merged**